docs/src/components/CodePreview/CodePreview.component.tsx

```diff
@@ -11,6 +11,9 @@ import {
     LiveError,
 } from 'react-live';
 import Component from '@reach/component-component';
+import Highlight, {defaultProps} from 'prism-react-renderer';
```

Comment:
I don't think this has been linted.

Reply:
Even with the ignore, there should be spaces on each side of the `defaultProps` import.

Reply:
Odd, I did run `ci`. Not sure why this didn't get picked up. Regardless I fixed it.